### PR TITLE
[RW-7278][risk=low]Remove PPW feature flag

### PR DESCRIPTION
**Pull request by yonghaoy**
Description:
Remove PPW feature flag first.
The next step would be cleaning up billing buffer code

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: api/config/config_local.json [hunk 1]
```diff
@@ -101,7 +101,6 @@
     "enableEventDateModifier": false,
     "enableResearchPurposePrompt": false,
     "enableGenomicExtraction": true,
-    "enableFireCloudV2Billing" : true,
```
Comment (calbach) [inline api/config/config_local.json:104]: Feature flag removal should happen in two steps: the config should be left in until the following release, see https://docs.google.com/document/d/133JzAcrIf_ZKFqzL2jvVaBbxb69NMMSjs7iTuj8xW0A/edit#bookmark=id.d3m7pc10ouq for details

**Review by calbach — COMMENTED** [review]
Thanks, just one process comment

File: api/config/config_local.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
     "leoBaseUrl": "https:\/\/leonardo.dsde-dev.broadinstitute.org",
     "xAppIdValue": "local-AoU-RW",
     "timeoutInSeconds": 60,
-    "jupyterDockerImage": "broadinstitute/terra-jupyter-aou:2.0.2",
+    "jupyterDockerImage": "broadinstitute/terra-jupyter-aou:2.0.1",
```
Comment (calbach) [inline api/config/config_local.json:11]: Missed in another PR?

**Review by calbach — APPROVED** [review]

File: api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java [hunk 3]
```diff
@@ -59,7 +56,7 @@
   private final AccessTierService accessTierService;
   private final BillingProjectBufferEntryDao billingProjectBufferEntryDao;
   private final Clock clock;
-  private final FireCloudService fireCloudService;
+  public final FireCloudService fireCloudService;
```
Comment (calbach) [inline api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java:59]: Why?

Comment (calbach) [reply api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java:59]: Even if this was done for testing, please find a different way. This should not be necessary

File: api/config/config_local.json (hunk 2 above)
Comment (yonghaoy) [reply api/config/config_local.json]: nope, just forgot to sync to head

File: api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java (hunk 3 above)
Comment (yonghaoy) [reply api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java]: intellij did that. Reverted.

**Pull request merged**